test/junit/scala/reflect/FieldAccessTest.scala

```diff
@@ -8,18 +8,17 @@ class FieldAccessTest {
 
   class TestClass {
     private val x = 123
-    // Uncommenting the following line would make the test fail
-    () => x
+    def fn = () => x
```

Comment:
I checked using the 2.11.12 REPL, this alternate form of the test does still fail in 2.11.x